### PR TITLE
Add min API auth to TOC /1

### LINKED ISSUE #27379: Hook up to TOC
@captainsafia  please review the two places where I'll put this article.

image

image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 3a9d7eb8-6c1f-4619-00fa-9b69dbe3dcea
* Version Independent ID: 3a9d7eb8-6c1f-4619-00fa-9b69dbe3dcea
* Content: [Configuring authentication for minimal APIs](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis/security?view=aspnetcore-7.0&branch=pr-en-us-27356)
* Content Source: [aspnetcore/fundamentals/minimal-apis/security.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/minimal-apis/security.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @captainsafia
* Microsoft Alias: **safia**

---
[Associated WorkItem - 33268](https://dev.azure.com/msft-skilling/Content/_workitems/edit/33268)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #27379
[Internal review URL](https://buildapi.docs.microsoft.com/Output/PullRequest/332f35cd-658d-4914-0f85-907e40ee89f0/202211040233047417-27496/BuildReport?accessString=0c960d6c1567ab36bc48d70dc422c1097c81fda88b1397a3f7359ada661389c6)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
cc @captainsafia

**Comment by Rick-Anderson** [comment]
@tdykstra I still think it should be in the Authz menu as it covers both authN and authZ. But I think folks will be able to find it now.

File: aspnetcore/fundamentals/minimal-apis/handle-errrors.md [hunk 1]
```diff
@@ -156,3 +156,7 @@ app.Run();
 ```
 
 For more information on using `AddProblemDetails`, see [Problem Details](/aspnet/core/fundamentals/error-handling?view=aspnetcore-7.0&preserve-view=true#pds7)
+
+## Additional Resources
+
+* <xref:fundamentals/minimal-apis/security>
```
Comment (captainsafia) [inline aspnetcore/fundamentals/minimal-apis/handle-errrors.md:162]: Hmmmm...why do we want to link to the security doc from here?

File: aspnetcore/fundamentals/minimal-apis/security.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Configuring authentication for minimal APIs
+title: Authentication and authorization in minimal APIs
```
Comment (captainsafia) [inline aspnetcore/fundamentals/minimal-apis/security.md:2]: Nice adjustment!

File: aspnetcore/toc.yml [hunk 3]
```diff
@@ -1245,6 +1245,8 @@ items:
             uid: security/authentication/index
           - name: Introduction to Identity
             uid: security/authentication/identity
+          - name: Authentication and authorization in minimal APIs
```
Comment (captainsafia) [inline aspnetcore/toc.yml:1248]: Can we put this under the minimal APIs heading instead?

Comment (Rick-Anderson) [reply aspnetcore/toc.yml:1248]: > Can we put this under the minimal APIs heading instead?

Sure, but it's a security topic and we don't do that for any other security topics. That's why I plastered all the Min API topics with a link the article. cc @tdykstra

File: aspnetcore/fundamentals/minimal-apis/handle-errrors.md (hunk 1 above)
Comment (Rick-Anderson) [reply aspnetcore/fundamentals/minimal-apis/handle-errrors.md]: reverted

**Comment by Rick-Anderson** [comment]
@captainsafia  see [this commit](https://github.com/dotnet/AspNetCore.Docs/pull/27496/commits/5443f9f3ced559bda2ed16deefd0d7c2f93a3f78)

File: aspnetcore/toc.yml [hunk 4]
```diff
@@ -720,6 +720,8 @@ items:
         uid: fundamentals/minimal-apis/middleware
       - name: Handle errors
         uid: fundamentals/minimal-apis/handle-errors
+      - name: Authentication and authorization in minimal APIs
```
Comment (tdykstra) [inline aspnetcore/toc.yml:723]:
```suggestion
      - name: Authentication and authorization
```

**Review by tdykstra — APPROVED** [review]
LGTM, one suggestion

**Review by captainsafia — APPROVED** [review]

**Pull request merged**